k8s-deployment/README.md

```diff
@@ -1,7 +1,7 @@
 
 # Installing the encoding service using helm
 
-## Add the N1 Analytics chart repo
+## Add the CSIRO's Data61 chart repo
```

Comment:
there are a few more N1 references in this file.

Reply:
Thanks - got em all now